app/views/school_groups/_sub_nav.html.erb

```diff
@@ -0,0 +1,33 @@
+<nav class="navbar navbar-expand-lg navbar-custom fixed-top fixed-top-sub-nav sub-navbar">
+  <div class="container">
+    <div>
+    </div>
+    <div>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#subnavbarNavDropdown" aria-controls="subnavbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="subnavbarNavDropdown">
+        <ul class="navbar-nav">
+          <li class="nav-item dropdown mt-2">
+            <a class="nav-link dropdown-toggle" data-toggle="dropdown" id="school-list-menu" href="#" role="button" aria-haspopup="true" aria-expanded="false"><%= t('school_groups.sub_nav.dashboards') %></a>
+            <div class="dropdown-menu scrollable" aria-labelledby="school-list-menu" id="manage_school_menu">
```

Comment:
Needs renaming? E.g. school-list for the link and school-list-menu for the drop-down?